.github/actions/height-integration/sync-issue/issue-opened/action.yml

```diff
@@ -0,0 +1,100 @@
+            "description": "${{ github.event.issue.description }}",
+            "listIds": [
+            "0540b3b8-bd44-4904-a97f-fd9d8780ef0a"
+            ],
+            "parentTaskId": "2dded224-8da0-4407-868f-d9cd018cbe52",
+            "status": "backLog",
+            "orderIntent": {
+            "intent": "end"
+            },
+            "assigneesIds": [
+            "848a5626-451f-4e15-b711-bd0a84883718",
+            "fe99c0a6-ad7b-458d-ae9f-687af57e0053",
+            "b0093618-6032-4e90-aa01-1dfc6628832d"
+            ],
+            "links": [],
+            "fields": [
+            {
+            "fieldTemplateId": "f1bae626-c18c-4198-b354-560d1610766c",
+            "name": "Tags",
+            "type": "labels",
+            "value": null,
+            "label": null,
+            "values": null,
+            "date": null,
+            "recursion": null,
+            "labels": [
+            {
+            "id": "a7ba7d3e-d1d6-4111-92e6-e89c9456ea13",
+            "model": "fieldLabel",
+            "value": "github-issue",
+            "date": null,
+            "hue": 49,
+            "deleted": false,
+            "archived": false
+            }
+            ],
+            "linkedTasks": null,
+            "linkedTimer": null,
+            "selectValue": null
+            }
+            ]
+            }'
+  # getIntakeTask:
+  #   runs-on: ubuntu-latest
+  #   # Map a step output to a job output
+  #   outputs:
+  #     intakeTask: # the output from createHeightTaskRequest step
+  #       id: ${{ steps.intakeTaskRequest.output.id }}
+  #   steps:
+  #     - id: intakeTaskRequest
+  #       name: Get the Github Issues Intake task object in Height
+  #       uses: fjogeleit/http-request-action@v1
+  #       with:
+  #         url: 'https://api.height.app/tasks/2925'
+  #         method: 'GET'
+  #         customHeaders: '{ "Content-Type": "application/json", "Authorization": "api-key ${{ secrets.HEIGHT_SECRET_TOKEN }}" }'
+  # moveHeightTaskToIntake:
+  #   runs-on: ubuntu-latest
+  #   needs: getIntakeTask
+  #   steps:
+  #     - id: moveHeightTaskToIntakeRequest
+  #       name: Move the new task into 'Github Issues Intake' in Height
+  #       uses: fjogeleit/http-request-action@v1
+  #       run: echo ${{ needs.createHeightTask.output.taskCreated.id }}
+  #       # with:
+  #       # Use ID of created task from previous step
+  #       # ${{ needs.createHeightTask.outputs.taskCreated }}
+  #       # url: 'https://api.height.app/tasks/${{ needs.createHeightTask.outputs.taskCreated.id }}'
+  #       # method: 'PUT'
+  #       # customHeaders: '{ "Content-Type": "application/json", "Authorization": "api-key ${{ secrets.HEIGHT_SECRET_TOKEN }}" }'
+  #       # Avara Task IDs:
+  #       # Github Issues Intake: 2dded224-8da0-4407-868f-d9cd018cbe52 (T-2925)
+  #       # data: '{ "name": "${{ github.event.issue.title }}", "listIds": ["64fa570a-a252-4521-b801-9b0eb18113bc"], "status": "359d42b4-5e7b-46fa-8465-d179b5a097ef" }'
```

Comment:
I'm going to remove this before merging, but testing out `GET`ting and then storing it as a var that can be used in the subsequent step to then update it. We'll have to do a GET then PUT to update existing tasks, so this is a future pattern that I'm testing.